README.md

```diff
@@ -1,6 +1,16 @@
-This is an experimental repository to split the SecureDrop development
-documentation from the main documentation in https://docs.securedrop.org/
+SecureDrop is an open-source whistleblower submission system that media organizations can use to securely accept documents from, and communicate with anonymous sources. It was originally created by the late Aaron Swartz and is currently managed by the [Freedom of the Press Foundation](https://freedom.press).
 
-It exists for review & collaboration purposes and is being experimentally
-built at https://developers.securedrop.org/. It should not be
-regarded as canonical.
\ No newline at end of file
+This repository is used to build the [developer documentation](https://developers.securedrop.org/) for SecureDrop.
```

Comment:
Another sentence like "Documentation for SecureDrop administrators is available at ______" would be nice too in case people get lost (or this starts doing well for searches of "securedrop docs").

Reply:
Done.

Reply:
Did you forget to commit or push this part? I don't see it.

Reply:
Was on CONTRIBUTING, now added to README as well